assets/javascripts/redmine_advanced_messenger.js

```diff
@@ -0,0 +1,75 @@
+const PWA_START_URL = "/my/page";
+const PWA_BADGE_VALUE = "PWA_BADGE_VALUE";
+const TIMER_CLOSING_OLD_NOTIFICATIONS = 5; // seconds
+
+// register service worker
+function registerPWAServiceWorker() {
+    if (navigator.serviceWorker) {
+        window.addEventListener("load", async function () {
+            try {
+                navigator.serviceWorker.register("../../plugin_assets/redmine_advanced_messenger/pwa/service-worker.js")
+                    .then((registration) => {
+                        console.log("Service worker registration succeeded:", registration);
```

Comment:
I'm curious where is shown this message ...

Reply:
I don't understand.